caasp-multi-master-stack.yaml

```diff
@@ -104,6 +117,48 @@ parameters:
     description: Root Password for the VMs
     default: linux
 
+  cpi_auth_url:
+    type: string
+    description: Cloud Provider Interface - Keystone API URL
+    default: https://api.keystone.example.net:5000/v3
+  cpi_domain_name:
+    type: string
+    description: Cloud Provider Interface - Keystone domain name
+    default: Default
+  cpi_tenant_name:
+    type: string
+    description: Cloud Provider Interface - Keystone project name
+    default: admin
+  cpi_region:
+    type: string
+    description: Cloud Provider Interface - Keystone region
+    default: region1
+  cpi_username:
+    type: string
+    description: Cloud Provider Interface - Keystone username
+    default: admin
+  cpi_password:
+    type: string
+    description: Cloud Provider Interface - Keystone user password
+  cpi_monitor_max_retries:
+    type: number
+    description: Cloud Provider Interface - Neutron Load balancer monitoring max retries
+    default: 3
+  cpi_bs_version:
+    type: string
+    description: Cloud Provider Interface - Cinder block storage API version
+    default: auto
+    constraints:
+      - allowed_values:
+          - v1
+          - v2
+          - v3
+          - auto
+  cpi_ignore_volume_az:
+    type: string
+    description: Cloud Provider Interface - Ignore Nova and Cinder availability zone
+    default: true
```

Comment:
Please put this string into quotation marks `"true"`

Reply:
done